plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -233,6 +255,8 @@ protected void copyTableSchema(Connection connection, String catalogName, String
         switch (jdbcTypeName) {
             case "varbinary":
                 return Optional.of(varbinaryColumnMapping());
+            case "datetimeoffset":
+                return Optional.of(timestampWithTimeZoneColumnMapping(typeHandle.getRequiredDecimalDigits()));
```

Comment:
Does unspecified precision mean the max precision? i.e. `datetimeoffset` == `datetimeoffset(7)`?

Reply:
Yes. https://docs.microsoft.com/en-us/sql/t-sql/data-types/datetimeoffset-transact-sql?view=sql-server-ver15
> The default fractional seconds precision is 100ns (seven digits for the fractional part of the seconds).